pyproject.toml

```diff
@@ -178,6 +178,8 @@ select = [
 extension-pkg-allow-list = [
     "numpy",
     "qiskit._accelerate",
+    "qiskit._qasm2",
```

Comment:
I'm surprised pylint wasn't complaining about this already.

Reply:
Looks like I actually tried it before, but it doesn't get all the way:
https://github.com/Qiskit/qiskit/blob/43ea026baba15f3496bef735aa28b5c5801ec3be/qiskit/qasm2/__init__.py#L529-L534